### PR TITLE
Language Select design v2

### PR DESCRIPTION
**Pull request by mateoclarke**
_Part of https://github.com/cityofaustin/techstack/issues/180_

screen shot 2018-03-20 at 6 28 44 pm

screen shot 2018-03-20 at 6 28 55 pm

### CODE REVIEW

**Comment by ifsimicoded** [comment]
Is the intent for the links to change position depending on which one is selected. 
screen shot 2018-03-20 at 7 07 46 pm
screen shot 2018-03-20 at 7 07 39 pm

File: src/js/page_sections/LanguageSelectBar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-static';
+import { find } from 'lodash';
+import { SUPPORTED_LANGUAGES } from 'js/i18n/constants';
+
+const selectedLanguage = (lang) => find(SUPPORTED_LANGUAGES, {'code': lang});
+
+const LanguageSelectBar = ({path, lang}) => (
+  <div className="coa-LanguageSelectBar">
+    <div className="wrapper">
```
Comment (ifsimicoded) [inline src/js/page_sections/LanguageSelectBar.js]: just an FYI, container-fluid is a class that auto adds 2rems of padding to the left and right of a container div.
I've been using it with wrapper to always ensure there's the same size padding on the left and right of container divs and same size max-widths.
So like
`<div class="container-fluid wrapper"> ...</div>`

Comment (mateoclarke) [reply src/js/page_sections/LanguageSelectBar.js]: 👍 👍 👍

**Comment by cthibaultatx** [comment]
@mateoclarke seeing your question above and I believe we covered it in issue [#207](https://github.com/cityofaustin/techstack/issues/207). The words should always stay in the same position- they are prioritized by number of people speaking each language at home,

1) Spanish
2) Vietnamese - next in line behind Chinese which we are building later
3) Arabic
4) In last position because it's highest spoken and default.

Important to keep user controls in the same place to reduce a person's memory load

**Pull request merged**